src/metrics.ts

```diff
@@ -0,0 +1,21 @@
+import { IncomingMessage, ServerResponse } from 'http'
+
+export async function serveMetrics(m: Metrics, res: ServerResponse) {
+  res.setHeader('content-type', 'text/plain; version=0.0.4')
```

Comment:
Oh you're serving these up like prometheus style?

Reply:
Yeah it used to be part of `next-boost` by default so I thought let's keep them around